templates/community_post_page.hbs

```diff
@@ -174,9 +182,17 @@
               <div class="comment-actions-container">
                 {{#unless official}}
                   <div class="comment-vote vote" role='radiogroup'>
-                    {{vote 'up' role='radio' class='vote-up' selected_class='vote-voted'}}
+                    {{#vote 'up' role='radio' class='vote-up' selected_class='vote-voted'}}
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16" transform="scale(1,-1)">
+                      <path fill="none" stroke="currentColor" stroke-linecap="round" stroke-width="2" d="M4 6.5l3.6 3.6c.2.2.5.2.7 0L12 6.5"/>
+                    </svg>
+                    {{/vote}}
```

Comment:
Can the `vote` helper accept a block already?

Reply:
> This changes should follow the following change in Help Center: zendesk/help_center#19781